pegasus/router.rb

```diff
@@ -316,7 +316,7 @@ def self.load_configs_in(dir)
   not_found do
     status 404
     path = resolve_template('views', settings.template_extnames, '/404')
-    document path
+    (document path).tap{dont_cache}
```

Comment:
ah, I see, not_found is not actually a route (which can call dont_cache in the main block). What you have is fine with me.

I might prefer document(path).tap since it seems more natural to use the method call parens instead of precedence parens.